Documents that setup timeout accepts only positive values (#1720)
grafana/k6-docs: docs/sources/next/using-k6/k6-options/reference.md

## Setup timeout

> Specify how long the `setup()` function is allow to run before it's terminated and the test fails. If defined the value must be positive.

Reviewer: ```suggestion
Specify how long the `setup()` function can run before it's terminated and the test fails. If defined, the value must be positive.
```
